index.js

```diff
@@ -215,6 +209,12 @@ CB.use(shortcodes({
 }));
 CB.use(timer('CB: Shortcodes'));
 
+// Don't rebuild files that have not been touched
+if (process.env.NODE_ENV === 'development') {
+  CB.use(revision);
```

Comment:
this looks at the file in its current state in the pipeline and removes it from further processing in case it looks identically (reasoning: "nothing changed, so we don't need to run all the processors on every file every time")

i arbitrarily placed that after shortcodes, so changes to shortcode-replacements do invalidate the cache (also the next processor - markdown - is the first one to be quite slow).